src/common/logSubscriptionStatus.ts

```diff
@@ -0,0 +1,20 @@
+import { logErrorReturnLogId, logInfoReturnLogId } from "@/logger/logger";
+
+export const checkAndLogSubscriptionStatus = (
+    status: "TIMED_OUT" | "CHANNEL_ERROR" | "CLOSED" | "SUBSCRIBED",
+    err: Error | undefined,
+    tableName: string
+): boolean => {
+    if (status === "TIMED_OUT") {
+        void logErrorReturnLogId(`Channel Timed Out: Subscribe to ${tableName} table`, err);
+        return false;
+    } else if (status === "CHANNEL_ERROR") {
+        void logErrorReturnLogId(`Channel Error: Subscribe to ${tableName} table`, err);
+        return false;
+    } else if (status === "CLOSED") {
+        void logInfoReturnLogId(`Subscription to ${tableName} table closed`);
+    } else {
+        void logInfoReturnLogId(`Subscribed to ${tableName} table`);
+    }
+    return true;
```

Comment:
nitpick: If we're simply going through every case of `status` then it feels nicer to use swtich case - but it might just be me 
```suggestion
    switch (status) {
        case "TIMED_OUT":
            void logErrorReturnLogId(`Channel Timed Out: Subscribe to ${tableName} table`, err);
            return false;
        case "CHANNEL_ERROR":
            void logErrorReturnLogId(`Channel Error: Subscribe to ${tableName} table`, err);
            return false;
        case "CLOSED":
            void logInfoReturnLogId(`Subscription to ${tableName} table closed`);
            return true;
        case "SUBSCRIBED":
            void logInfoReturnLogId(`Subscribed to ${tableName} table`);
            return true;
    }
```

Reply:
Agreed